backend/.eslintrc.json

```diff
@@ -10,11 +10,9 @@
   ],
   "parser": "@typescript-eslint/parser",
   "parserOptions": {
-    "ecmaVersion": 12,
+    "ecmaVersion": "latest",
```

Comment:
This is the default setting from `npx eslint init` - I think this is should be fine to leave as `latest` so that the ESLint config is always ready to parse the latest ECMAScript features

Reply:
Aight yeah it's probably less problematic than node